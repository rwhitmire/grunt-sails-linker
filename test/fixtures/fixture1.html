<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title></title>
  <link rel="stylesheet" href="">
</head>
<body>
  
  <!--SCRIPTS src="fixtures/games/*.js"-->
  <!--SCRIPTS END-->

  <!--SCRIPTS src="fixtures/languages/*.js"-->
  <!--SCRIPTS END-->

</body>
</html>