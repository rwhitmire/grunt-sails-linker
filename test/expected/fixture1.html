<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title></title>
  <link rel="stylesheet" href="">
</head>
<body>
  
  <!--SCRIPTS src="fixtures/games/*.js"-->
  <script src="fixtures/games/everquest.js"></script>
  <script src="fixtures/games/minecraft.js"></script>
  <!--SCRIPTS END-->

  <!--SCRIPTS src="fixtures/languages/*.js"-->
  <script src="fixtures/languages/csharp.js"></script>
  <script src="fixtures/languages/rust.js"></script>
  <!--SCRIPTS END-->

</body>
</html>